### PR TITLE
add srreal

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@pavoljuhas thoughts?

File: recipes/srreal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% set name = "srreal" %}
+{% set version = "v1.2" %}
```
Comment (pavoljuhas) [inline recipes/srreal/meta.yaml]: Replace with "1.2", the `v` is in tag name, but package version should not include it.

File: recipes/srreal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/diffpy/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (pavoljuhas) [inline recipes/srreal/meta.yaml]: Might be better to use a sdist tarball from pypi.io
https://pypi.org/project/diffpy.srreal/#files

**Comment by pavoljuhas** [comment]
> thoughts?

circle-ci failed, because it did not have compiler paths set, toolchain should fix that:
```
git pull https://github.com/pavoljuhas/staged-recipes.git srreal-fixups
```

travis could not resolve diffpy.structure.  Is there a package for Python 3.6 on conda-forge?

File: recipes/srreal/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{% set name = "srreal" %}
```
Comment (pavoljuhas) [inline recipes/srreal/meta.yaml]: Package name should be "diffpy.srreal".

**Pull request closed without merging**